sysl2/sysl/seqs.go

```diff
@@ -147,34 +135,36 @@ func DoGenerateSequenceDiagrams(stdout, stderr io.Writer, flags *flag.FlagSet, a
 			log.Errorln(err)
 		}
 	}()
-	var endpoints_flag, apps_flag, blackboxes_flag arrayFlags
-	root := flags.String("root", ".", "sysl root directory for input model file (default: .)")
-	endpoint_format := flags.String("endpoint_format", "%(epname)", "Specify the format string for sequence diagram endpoints. "+
-		"May include %%(epname), %%(eplongname) and %%(@foo) for attribute foo(default: %(epname))")
-	app_format := flags.String("app_format", "%(appname)", "Specify the format string for sequence diagram participants. "+
-		"May include %%(appname) and %%(@foo) for attribute foo(default: %(appname))")
-	title := flags.String("title", "", "diagram title")
-	plantuml := flags.String("plantuml", "", strings.Join([]string{"base url of plantuml server",
+	sd := kingpin.New("sd", "Generate sequence diagram")
+	root := sd.Flag("root", "sysl root directory for input model file (default: .)").Default(".").String()
+	endpoint_format := sd.Flag("endpoint_format", "Specify the format string for sequence diagram endpoints. "+
+		"May include %%(epname), %%(eplongname) and %%(@foo) for attribute foo(default: %(epname))").Default("%(epname)").String()
+	app_format := sd.Flag("app_format", "Specify the format string for sequence diagram participants. "+
+		"May include %%(appname) and %%(@foo) for attribute foo(default: %(appname))").Default("%(appname)").String()
+	title := sd.Flag("title", "diagram title").Short('t').String()
+	plantuml := sd.Flag("plantuml", strings.Join([]string{"base url of plantuml server",
```

Comment:
is it possible to set the default value of plantuml here?

Reply:
It's possible. But I get another question. After setting default value here, I believe part of the code in diagutil.go should be changed too.
if plantuml == "" {
	plantuml = os.Getenv("SYSL_PLANTUML")
} If better to remove this, too?